package/scie-pants.lift.json

```diff
@@ -92,9 +92,10 @@
             "description": "Runs a hermetic Pants installation.",
             "env": {
               "=PANTS_VERSION": "{scie.bindings.configure:PANTS_VERSION}",
-              "PANTS_BUILDROOT_OVERRIDE": "{scie.bindings.configure:PANTS_BUILDROOT_OVERRIDE}"
+              "PANTS_BUILDROOT_OVERRIDE": "{scie.bindings.configure:PANTS_BUILDROOT_OVERRIDE}",
+              "=_PANTS_SERVER_EXE": "{scie.bindings.install:PANTS_SERVER_EXE}"
```

Comment:
My brain would like these sorted.. no matter otherwise.

Reply:
Technically they are sorted, due to the prefix `_`? But I don't think that it will begin to matter until/unless there are more of them.